app/src/test/groovy/gov/hhs/cdc/trustedintermediary/OrdersMock.groovy

```diff
@@ -0,0 +1,33 @@
+package gov.hhs.cdc.trustedintermediary
+
+import gov.hhs.cdc.trustedintermediary.etor.demographics.LabOrder
+/**
+ * A mock implementation of the {@link gov.hhs.cdc.trustedintermediary.etor.demographics.LabOrder} interface that is easy to use in tests.
+ */
+class OrdersMock<T> implements LabOrder<T> {
```

Comment:
Perhaps call this `LabOrderMock`?  That way its similar to the `DemographicsMock`.